src/Money.php

```diff
@@ -1,42 +1,26 @@
 <?php
+
 namespace Ayeo\Price;
 
 class Money
```

Comment:
To me such money concept is worthless. It does not reflect any real knowledge. I think money consist of float value and currency. The 100.0 is not money at all. I can not see any advantages of introducing such idea here